internal/test/wsu/main_test.go

```diff
@@ -8,12 +8,15 @@ import (
 	"testing"
 )
 
-// framework holds the instantiation of test suite being executed. As of now, temp dir is hardcoded.
 var (
-	framework = &e2ef.TestFramework{}
+	wsuFramework = WsuFramework{}
 	// TODO: expose this to the end user as a command line flag
 	// vmCount is the number of VMs the test suite requires
-	vmCount = 1
+	// Bring up 2 vms to test wsu run with automatic download of WMCB based on cluster version as well as built from
+	// source version
+	vmCount = 2
+	// Number of VMs which should use built version of WMCB
+	vmCountWithBuiltWmcb = 1
```

Comment:
Shouldn't this be a property of the `WsuFramework`?

Reply:
I can change to make it so - I figured keeping this option here (and eventually expose it to command line) would be a nice-to-have

Reply:
You would need to separate out the two concerns, i.e. command line option vs framework and then use what is giving in the CLI to instantiate the framework

Reply:
The variable will be mapped to a field in `WsuFramework` so that endusers can access it via cli. Correct me if I am wrong @suhanime

Reply:
Correct @ravisantoshgudimetla . For now I am separating the concerns though - I moved the variable to be a property of wsuFramework and hardcoding it to 1, but the building blocks are in place if we wanted to expose it to CLI (with vmCount as well) in the future